addons/im_odoo_support/static/src/js/im_odoo_support.js

```diff
@@ -44,6 +44,9 @@
             }
         },
         start_support: function(){
+            // The free livechat service doesn't exist anymore
+            window.open('https://www.odoo.com/help', '_blank');
+            return;
```

Comment:
IMHO:
- In master, remove the module
- In 9.0, change the manifest since the "Support" menu is defined in `web`: https://github.com/odoo/odoo/blob/9.0/addons/web/static/src/js/widgets/user_menu.js#L62
- In 8.0 however, `im_odoo_support` defines the "Odoo Support" button, and I assume we want to keep it.

As I mentioned before, the idea of this diff was to avoid "cleaning" something which is ultimately intended to disappear.